components/InputLabel.tsx

```diff
@@ -16,11 +16,12 @@ const InputLabel: FC<InputLabelProps> = ({
   textRequired,
 }) => {
   return (
-    <label
-      id={id}
-      htmlFor={htmlFor}
-      className={`mb-2 block font-bold ${required ? 'required' : ''}`}
-    >
+    <label id={id} htmlFor={htmlFor} className="mb-2 block font-bold">
+      {required && (
+        <b className="text-accent-error mr-0.5" aria-hidden="true">
```

Comment:
Change this html to this:

```tsx
<span className="text-accent-error" aria-hidden="true">
  {'* '}
</span>
```
`'* '` ensure the correct space between the asterisk and the label's text and `b` tag is not needed because the label have the bold class.